boule/_triaxialellipsoid.py

```diff
@@ -100,6 +101,10 @@ class TriaxialEllipsoid:
 
     >>> print(f"{ellipsoid.mean_radius:.0f} m")
     262700 m
+    >>> print(f"{ellipsoid.area:.10e} m²")
+    8.6562393883e+11 m²
+    >>> print(f"{ellipsoid.area_equivalent_radius:0.0f} m")
+    262458 m
```

Comment:
@MarkWieczorek out of curiosity, where did these numbers come from?

Reply:
That number comes from the output of the boule command.